package.json

```diff
@@ -4,27 +4,30 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "dev": "./node_modules/.bin/nodemon index.js",
+    "dev": "./node_modules/.bin/env-cmd ./node_modules/.bin/nodemon index.js",
+    "dev:migrate": "./node_modules/.bin/env-cmd ./node_modules/.bin/sequalize db:migrate --env development --config ./config/sequelize.js",
```

Comment:
You've got two typos that keep your migrations from running properly. The first is you have `sequalize`. The second is you have the path of your config file as `./config/sequelize.js` but your folder is actually names `configs`